Sources/hostmgr/main.swift

```diff
@@ -18,25 +19,6 @@ struct Hostmgr: ParsableCommand {
             BenchmarkCommand.self,
             ConfigCommand.self
         ])
-
-    @Flag(help: "Print the version and exit")
-    var version: Bool = false
-
-    func run() throws {
-        guard version == false else {
-            print(appVersion)
-            return
-        }
-
-        logger.debug("Starting Up")
-
-        guard Configuration.isValid else {
-            print("Invalid configuration – exiting")
-            throw ExitCode(1)
-        }
-
-        throw CleanExit.helpRequest(self)
-    }
```

Comment:
> So, no CLI is changed, but we have less code.

This is pretty cool, thank you for implementing it.

A nice benefit of using the default `run()` implementation is that calling `hostmgr` with no arguments now prints the help, as per [the docs](https://apple.github.io/swift-argument-parser/documentation/argumentparser/parsablecommand/run()-7p2fr/).

```
# trunk
➜ swift run --skip-build hostmgr
2022-08-31T13:22:14+1000 debug com.automattic.hostmgr : Starting Up
Invalid configuration – exiting

# this branch
➜ swift run --skip-build hostmgr
OVERVIEW: A utility for managing VM hosts

USAGE: hostmgr <subcommand>

OPTIONS:
  --version               Show the version.
  -h, --help              Show help information.

SUBCOMMANDS:
  vm                      Allows working with VMs
  sync                    Sync remote data with this host
  init                    Create a configuration file
  generate                Data processing tasks
  set                     Set system values
  benchmark               System tests
  config                  Display the system configuration

  See 'hostmgr help <subcommand>' for detailed help.
```